### ISSUE TITLE
Growth without growth hormone

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6659]
**Preferred term label:**
Growth without growth  hormone

**Synonyms**
Growth without GH

**Definition (free text, please give PubMed ID)**
Maintaining normal linear growth despite abnormal GH secretion. 
PMID: 28443260         

**Parent term (use hpo.jax.org/app)**
Growth abnormality

**Comment by pnrobinson** [comment]
adding as HP_0033795
